atlasdb-api/src/main/java/com/palantir/atlasdb/persistent/api/PersistentStore.java

```diff
@@ -27,76 +27,76 @@
 
 import com.palantir.logsafe.exceptions.SafeIllegalArgumentException;
 
-public interface PersistentTimestampStore extends AutoCloseable {
+public interface PersistentStore extends AutoCloseable {
     @Value.Immutable
     interface StoreNamespace {
         String humanReadableName();
         UUID uniqueName();
     }
 
     /**
-     * Gets the commit timestamp associated with the entry specified by {@code startTs}.
+     * Gets the value associated with the entry specified by {@code key}.
      *
-     * @param storeNamespace handle to the namespace from which we want to retrieve the commit timestamp
-     * @param startTs        entry key for which we want to retrieve commit timestamp
-     * @return the associated timestamp or null if the entry is missing
-     * @throws com.palantir.logsafe.exceptions.SafeIllegalArgumentException when {@code storeNamespace} is a
-     * handle to a non existing namespace
+     * @param storeNamespace handle to the namespace from which we want to retrieve the value
+     * @param key            entry key for which we want to retrieve the value
+     * @return the associated value or null if the entry is missing
+     * @throws com.palantir.logsafe.exceptions.SafeIllegalArgumentException when {@code storeNamespace} is a handle to a
```

Comment:
can this be imported? it pushes everything to the far right

Reply:
Imported.